snapd-glib/snapd-snap.c

```diff
@@ -424,6 +426,13 @@ snapd_snap_get_hold (SnapdSnap *self)
 }
 
 
+GDateTime *
```

Comment:
Missing documentation

Reply:
Ops... you are right.

BTW: I see in all the documentation that it uses "@snap: a @SnapdSnap", but the parameter name in the function is `SnapdSnap *self`... Shouldn't it be "@self: a @SnapdSnap"?

Reply:
Fixed.

Reply:
Responded in the other PR - the C files use 'self' and the headers/docs use the specific name.